docs/products/metrics/howto/download-influxdb-data.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Download Aiven for InfluxDB data backup
+sidebar_label: Download InfluxDB data
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Download a backup of your Aiven for InfluxDB service data by requesting a backup. This creates a snapshot of your latest data, ready for download.
```

Comment:
I think we should add somewhere an explanation about the backup format. We create a `portable` backup, including all databases and retention policies. The backup is than packed into a single `.tar` file.  (Influxdb docs, if neceessary: https://docs.influxdata.com/influxdb/v1/administration/backup_and_restore/).

Reply:
This is a good point, thanks. I have added a section after the prerequisites.